### PR TITLE
Schedule datastore export

### PR DESCRIPTION
**Pull request by jlara310**
Submit sample code to update [Scheduling Datastore exports](https://cloud.google.com/datastore/docs/schedule-export) documentation to use Cloud Functions and Cloud Scheduler instead of App Engine.

### CODE REVIEW

**Comment by jlara310** [comment]
@BenWhitehead

**Comment by BenWhitehead** [comment]
Looks like you need to add a `requirements-test.txt` similar to [this one](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/datastore/cloud-client/requirements-test.txt) so that the tests can bootstrap and run in the failing Kokoro builds.

**Comment by tmatsuo** [comment]
@jlara310 Please add tests

**Comment by jlara310** [comment]
> @jlara310 Please add tests

Acknowledged. Working on adding some unit tests.

**Comment by jlara310** [comment]
@tmatsuo , added unit tests. I'm a bit confused about the new Nox set-up. Do I need to add a noxfile.py for my tests or is it generated for me?

File: datastore/schedule-export/requirements-test.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pytest==5.3.2
+gcp-devrel-py-tools==0.0.15
+flaky==3.6.1
```
Comment (tmatsuo) [inline datastore/schedule-export/requirements-test.txt:3]: I don't think you're using gcp-devrel-py-tools or flaky.

Comment (jlara310) [reply datastore/schedule-export/requirements-test.txt]: Removed unused imports, thanks!

**Comment by BenWhitehead** [comment]
@GoogleCloudPlatform/python-samples-owners Can we get someone to do a final once-over and merge this for @jlara310 ?

File: datastore/schedule-export/main.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+datastore = build('datastore', 'v1')
+project_id = os.environ.get('GCP_PROJECT')
+
+
+def datastore_export(event, context):
+    '''Trigger a Datastore export from a Cloud Scheduler job.
+    Args:
+         event (dict): event[data] must contain a json object encoded in
+         base-64. Cloud Scheduler encodes payloads in base-64 by default.
```
Comment (tmatsuo) [inline datastore/schedule-export/main.py:15]: NIT: I think you need additional indent here for continuous lines for argument description.

File: datastore/schedule-export/main.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+         Object must include a 'bucket' value and can include 'kinds'
+         and 'namespaceIds' values.
+         context (google.cloud.functions.Context): The Cloud Functions event
+         metadata.
```
Comment (tmatsuo) [inline datastore/schedule-export/main.py:19]: nit: ditto

**Comment by jlara310** [comment]
Thank you, both!

**Pull request merged**